index.js

```diff
@@ -2,32 +2,34 @@ import Punycode from 'punycode/punycode.js';
 
 import rules from './data/rules.js';
 
-
 var internals = {};
 
 
```

Comment:
Is there a specific reason for this whitespace here? It's such a tiny change but I'd just leave it like this
```suggestion

```

Reply:
No specific reason... it just slipped through...